src/ADAL.PCL.iOS/PlatformInformation.cs

```diff
@@ -76,5 +78,20 @@ public override string GetAssemblyFileVersionAttribute()
             // TODO: Check if assembly file version can be read in iOS assembly as well or not. For now, we use assembly version instead.
             return typeof(AdalIdHelper).GetTypeInfo().Assembly.GetName().Version.ToString();
         }
+
+        public override string GetApplicationName()
+        {
+            return NSBundle.MainBundle.InfoDictionary["CFBundleDisplayName"].ToString();
```

Comment:
This would return name of the app that can change. Please discuss with @jasoncoolmax or @RPangrle is we should be returning bundle id instead.